packages/core-rn/src/metrics/RNMetricsService.js

```diff
@@ -0,0 +1,28 @@
+import { NativeModules } from "react-native";
+import { MetricsService } from "@aerogearservices/core";
+import packageJson from "../../package.json";
+
+const RNMobileCore = NativeModules.MobileCore;
+
+class RNMetricsService extends MetricsService {
+
+  async sendAppAndDeviceMetrics() {
+    const deviceMetrics = await RNMobileCore.getDeviceMetrics();
+    const appMetrics = await RNMobileCore.getAppMetrics();
+
+    appMetrics.sdkVersion = packageJson.version + "-rn";
```

Comment:
@paolobueno @pb82 @david-martin 
How do you like this approach? It would be sth like `1.0.0-rn` `1.0.0-cordova`.

Reply:
Would it make sense the other way around? e.g. `react-native-1.0.0` and `cordova-1.0.0`

Also, are there changes to the android & ios sdk to conform to this too? e.g. `ios-1.0.0' ?

Reply:
Good point, right now it's omitted in iOS and Android. On the other hand I wonder if this is really necessary though, since the developer knows already what SDK he's using.

Reply:
> On the other hand I wonder if this is really necessary though, since the developer knows already what SDK he's using.

Big +1 on this point. Personally I would find something like `cordova-1.0.0` or `react-native-1.0.0` unintuitive and confusing, especially when it starts to surface in the dashboards. Also, if users want to build tooling/automation/decision making on top of the raw metrics data, then this format will cause some headaches.

I see two ways of understanding which SDK an app is using:

1. We could assume the developers/owners of the app already know. Perhaps this isn't the best assumption to make.
2. We explicitly add a new field to the App payload that identifies the SDK being used.